neofetch

```diff
@@ -973,6 +973,15 @@ get_distro() {
                 esac
                 distro=$(lsb_release "$lsb_flags")
 
+                # Temporary support for Regolith Linux
+                if [[ $DESKTOP_SESSION == "regolith" ]]; then
+                    case $distro_shorthand in
+                        on)   distro+="-REGOLITH" ;;
+                        tiny) distro+="-REGOLITH" ;;
+                        *)    distro=$(lsb_release -si)"-REGOLITH "$(lsb_release -sr) ;;
+                    esac
+                fi
+
```

Comment:
This should be removed and only the ASCII logo should be kept. Regolith is only a DE right? Even this term is pushing it as it seems to be a configuration on top of regular GNOME and i3(?)

I'm happy to include the logo as well as temporary support for DE detection but I'm unconvinced as to adding it as a Linux distribution. Typically when none of the existing detection methods work for a distribution there are two reasons.

1) The distribution is based on another distribution but is only a light "skin" on top of it. No additional repositories are added, no identification files are updated and it is in reality what it is based on.
2) The distribution is unique in that it doesn't follow FreeDesktop/Linux Standards Base conventions for Linux distribution identification. 

I'm reluctant to add special cases for every single distribution which falls under (1) as the list would be endless, hard-coded and require maintenance. That being said, exceptions are made for (1) within reason. 

For Regolith to be considered a distribution by the existing detection methods a simple `/etc/os-release` file must be added to the distribution. It's that simple. The other alternative is to provide the `lsb-release` program with a Regolith configuration file. Also see: https://www.linux.org/docs/man5/os-release.html

If this isn't possible for Regolith, it points back to (1).

Here's an example (`/etc/os-release`):

```
NAME="KISS Linux"
PRETTY_NAME="KISS Linux"
ID=kiss
BUILD_ID=rolling
HOME_URL="https://kisslinux.github.io"
DOCUMENTATION_URL="https://kisslinux.github.io"
SUPPORT_URL="https://github.com/kisslinux/packages/issues"
BUG_REPORT_URL="https://github.com/kisslinux/packages/issues"
```

Reply:
Yes, that's ok, I completely agree with you. I just failed to find another way for the logo to show up if only `de` is specified :thinking:

Reply:
The DE specific logos aren't automatically used. They're enabled manually by the user through their configuration file or at runtime using `--ascii_distro name_here`. 

This is due to the fact that there is no way to distinguish the DE as a Linux distribution. Again related to (1) in my above post. Ubuntu-GNOME is no different to Ubuntu with GNOME installed to give an example. 

The Ubuntu-GNOME project (and other Ubuntu "flavors") make no effort to identify as anything other than Ubuntu... as they *are* Ubuntu. Neofetch *is* right here and the issue is external.

(Simply adding an `os-release` file is all it takes for detection to work.)

Reply:
Understood. That makes sense. :+1:

Reply:
Fixed

Reply:
Hi @dylanaraps and @lawzava , thanks for taking the time to look into adding support for Regolith :smile: .  As you've discovered, Regolith makes no effort to distinguish itself from Ubuntu, which is intentional due to a Regolith design goal of being "brandless" at runtime.  I would classify Regolith as an unofficial, unsponsored, and unrecognized Ubuntu "respin".  If someone has the time and expertise to port other distributions I would support that but cannot do so given my limited time budget.  It's Ubuntu-only due to circumstance, not design.  However, it is a distinct desktop environment, based on gnome-flashback and i3.  It is neither easily containable by either "i3" or "gnome-flashback" but a hybrid of both.

I confess to not knowing much about the taxonomy of desktop environments, but if 'gnome' is the container for all gnome-based desktop environments, that seems like we're at the end of it.  Although to me, such a generalized classification isn't very useful.  gnome-flashback is very different in scope and capability than gnome-shell, not to mention MATE and other variants.

As the project lead, ideally I would like Regolith to be classified as a distinct DE (assuming that level of granularity is supported in software), but not as a distinct distribution.  (Regolith's distribution as ISO is intended mainly as a convenience to users that may be starting from scratch.)  I am happy to work with @lawzava to export the proper environment variables such that `neofetch` can consistently recognize it as such without any need for exceptional code paths.  From reviewing the dialog, it would seem this is the least-risk option as it's entirely new and is unlikely to cause regressions elsewhere:
> Provide $REGOLITH_DESKTOP_SESSION_ID or another environment variable unique to Regolith.

@dylanaraps is this the best way of identifying Regolith as a distinct DE or would you recommend something else?

Reply:
Thanks for getting in touch. That would be the best way, yes. :+1: